### PR TITLE
ERMS-3000-2

### PR DESCRIPTION
**Pull request by Inselhopper**

### CODE REVIEW

File: grails-app/views/platform/accessMethods.gsp [hunk 1]
```diff
@@ -51,11 +51,15 @@
...
-                                <g:link action="edit" controller="accessMethod" id="${accessMethod?.id}" class="ui icon button">
-                                    <i class="write icon"></i>
+                                <g:link action="edit" controller="accessMethod" id="${accessMethod?.id}" class="ui icon button"
+                                        role="button"
+                                        aria-label="${message(code: 'ariaLabel.edit.universal')}">
+                                    <i aria-hidden="true" class="write icon"></i>
                                 </g:link>
                                 <g:link action="delete" controller="accessMethod" id="${accessMethod?.id}" class="ui negative icon button"
-                                        onclick="return confirm('${message(code: 'accessPoint.details.delete.confirm', args: ['aaaa'])}')">
+                                        onclick="return confirm('${message(code: 'accessPoint.details.delete.confirm', args: ['aaaa'])}')"
```
Comment (klodav) [inline grails-app/views/platform/accessMethods.gsp:60]: args: ['aaaa'] ?

File: grails-app/views/subscription/linkLicenseMembers.gsp [hunk 2]
```diff
@@ -173,8 +173,8 @@
                         <g:link controller="subscription" action="show" id="${sub.id}" class="ui icon button"
                                 data-tooltip="${message(code:'subscription.details.viewMember.label')}"
                                 data-position="left center"
-                        >
-                            <i class="write icon"></i></g:link>
+                                role="button"
```
Comment (klodav) [inline grails-app/views/subscription/linkLicenseMembers.gsp:176]: Tag not closed; missing '>'

File: grails-app/views/platform/accessMethods.gsp (hunk 1 above)
Comment (croqueGrec09) [reply grails-app/views/platform/accessMethods.gsp:60]: That's the shout of the access method when it realizes that it is going to be deleted! :D

File: grails-app/views/templates/properties/_groupBindings.gsp [hunk 3]
```diff
@@ -102,7 +102,10 @@
                                                   ]'
                                                   onComplete="c3po.initProperties('${createLink(controller:'ajaxJson', action:'lookup')}', '#propDefGroupBindingConfig')"
                                                   data-update="propDefGroupBindingConfig"
-                                                  class="ui icon negative button">
+                                                  class="ui icon negative button"
+                                                      role="button"
+                                                      ariaLabel="${message(code: 'ariaLabel.delete.universal')}">
+                                    >
```
Comment (klodav) [inline grails-app/views/templates/properties/_groupBindings.gsp:108]: duplicate '>'  ?

Comment (Inselhopper) [reply grails-app/views/templates/properties/_groupBindings.gsp]: fixed

**Review by klodav — APPROVED** [review]

**Pull request merged**